ui-components/src/page_components/listing/AdditionalFees.stories.tsx

```diff
@@ -12,9 +12,14 @@ export const allFields = () => {
       depositMin={"1140"}
       depositMax={"1500"}
       applicationFee={"30"}
-      costsNotIncluded={
-        "Resident responsible for PG&E, internet and phone.  Owner pays for water, trash, and sewage.  Residents encouraged to obtain renter's insurance but this is not a requirement.  Rent is due by the 5th of each month. Late fee $35 and returned check fee is $35 additional."
-      }
+      footerContent={[
```

Comment:
Would like to see some stories added that showcase the new ability for the footer to have multiple content sections, probably as its own story.

Reply:
Added!